spec/unit/pops/types/p_object_type_spec.rb

```diff
@@ -687,6 +675,35 @@ def parse_object(name, body_string)
       expect(eval_and_collect_notices(code)).to eql(['true'])
     end
 
+    it 'a named object is added to the loader irrespective of alias declaration' do
+      code = <<-CODE
+      $x = Object[{ name => 'MyFirstObject' }]
+      notice($x == MyFirstObject)
+      CODE
```

Comment:
I think this is wrong - just because we have an instance of ObjectType does not mean that it is loaded/imported. I would expect the reference on line 681 to fail with undefined "MyFirstObject". The only way to get a name is to bind an object to a type with `type = ...` or when the object type is in a type set. All other instances may have a name, or be anonymous. They are useful when doing type algebra.

The rationale for this is that it must be possible to juggle ObjectType instances without having them pollute the name space.